.github/workflows/ccpp.yml

```diff
@@ -41,7 +41,8 @@ jobs:
            ../configure \
                --enable-languages=rust \
                --disable-bootstrap \
-               --enable-multilib
+               --enable-multilib \
+               --enable-selftest
```

Comment:
As far as I can tell, there is no such option in GCC, so that is a no-op, and should be removed.

Reply:
Done! Thanks for the keen eye, I don't know why I thought this was important